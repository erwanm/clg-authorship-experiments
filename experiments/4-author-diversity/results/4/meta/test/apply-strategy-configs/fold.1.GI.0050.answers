1903-us-tjoas 1854-hbs-smofl	0.500268
1895-wta-alae 1906-mt-cfma	0.88394
1891-fmc-ttf 1881-haj-fcbtp	0.88394
1872-haj-ptf 1906-rwc-ttolp	0.555079
1898-hj-ttm 1879-hj-h	0.418053
1874-wta-tcw 1900-haj-adoh	0.582484
1878-hj-te 1882-lma-ajsb-v6	0.418053
1883-hj-rh 1898-hj-itc	0.88394
1885-hj-gr 1910-us-sts	0.226217
1914-us-sm 1897-ewaoc-tdof	0.335838
1896-espw-cfal 1887-espw-jtf	0.88394
1887-espw-tgb 1900-espw-wtg	0.582484
1894-cdw-tgh 1907-fmc-tdr	0.555079
1900-fmc-itpotk 1905-fmc-fm	0.555079
1889-fmc-g 1898-haj-tyb	0.664699
1890-hj-ttm 1904-hj-tgb	0.801724
1878-hbs-pp 1870-hbs-tbc	0.555079
1901-us-km 1921-us-tbol	0.253622
1883-hj-rh-v2 1872-wta-lb	0.253622
1866-haj-ccc-ia 1871-hbs-mwai	0.555079
1880-hj-c 1885-hj-altof	0.88394
1898-fmc-ari-v2 1893-fmc-kl-v2	0.363243
1918-rwc-tlg 1863-wta-isao	0.006976
1884-ab-sbts 1898-us-apom	0.418053
1889-cfw-jl 1915-rwc-p	0.88394
1872-mt-ri 1895-haj-tdw	0.253622
1902-fmc-c 1883-fmc-tl	0.555079
1859-hbs-tmw 1887-fmc-pp	0.774319
1884-wta-Sac 1882-wta-aa	1
1871-haj-tt 1876-wta-vao	0.82913
1913-es-tm 1911-rwc-tgr	0.88394
1905-es-tpoe 1907-es-tlotg	0.692104
1899-haj-mmv 1866-haj-hf	0.390648
1891-fmc-do 1876-cdw-itl	0.500268
1892-fmc-pg 1883-fmc-dc	0.88394
1917-ab-bn 1922-us-tcmc	0.719509
1889-cdw-sths 1869-espw-mwag	0.664699
1902-hj-twotd 1896-hj-tsop	0.472863
1888-haj-bb 1899-haj-ra	0.664699
1913-ew-tcotc 1868-haj-rd	0.335838
1868-espw-tga 1919-rwc-tsos	0.801724
1900-haj-ainy 1919-ew-m	0.198812
1881-mt-tpatp 1899-haj-aoatb	0.390648
1891-fmc-k 1893-fmc-tr	0.88394
1881-hj-tpoal 1890-fmc-acmr	0.88394
1869-haj-mtmb 1903-haj-bba-ph	0.88394
1885-fmc-atoalp 1905-fmc-wso	1
1868-wta-mob 1870-wta-faf	0.664699
1871-hbs-pawt 1904-rwc-isotu	0.719509
1875-cfw-cn 1864-lma-m	0.582484
1914-mh+-mhce-nf 1908-fmc-tp	0.664699
1893-fmc-md 1895-fmc-ajs	1
1864-wta-tsb 1883-wta-sh	0.664699
1900-rwc-taoamm 1910-rwc-tgm	0.472863
1903-fmc-thor 1887-fmc-s	0.88394
1884-mt-taohf 1874-mt-lotm	0.719509
1907-espw-tldup 1871-espw-tsp	0.527673
1866-espw-gcj 1882-haj-bn	0.88394
1883-haj-tbb 1888-es-ttatv	0.774319
1908-fmc-s 1908-fmc-tws	0.88394
1889-wta-wtel 1911-rwc-tcl	0.061787
1904-rwc-sm 1886-hj-pc	0.692104
1918-us-jh 1897-ew-tdof	0.664699
1872-lma-ss 1867-lma-tagomtt	0.335838
1910-ew-tomag 1889-es-atg	0.719509
1889-espw-tsfi 1895-espw-tsojc	0.774319
1905-haj-fftf 1870-haj-rar	0.609889
1882-espw-dz 1909-mt-isd	0.390648
1907-ew-tfott 1896-hj-toh	0.88394
1868-wta-dad 1885-wta-sts	0.88394
1909-rwc-tdm 1897-ab-tdohy	0.856535
1891-haj-tetb 1868-haj-faf	0.88394
1893-haj-dtn 1866-haj-tcw	0.88394
1869-wta-dtr 1865-wta-waw	0.445458
1891-cdw-oi 1873-hbs-wish	0.88394
1879-hj-dm 1903-hj-ta	0.582484
1874-cfw-tac 1882-fmc-mi	0.774319
1893-espw-dm 1896-ab-boat	0.500268
1896-espw-tsasa 1894-espw-asl	0.664699
1879-haj-tym 1869-haj-pjb	0.88394
1920-us-1tsoap 1913-us-dg	0.445458
1917-es-ow-sh 1873-hbs-pl	0.82913
1855-wta-aa 1873-wta-tyc	0.390648
1879-haj-ttb 1894-fmc-wg	0.034382
1898-fmc-vc 1895-fmc-t	0.88394
1867-espw-gyatgc 1901-espw-a	0.555079
1896-fmc-c 1885-fmc-zamc	0.88394
1855-hbs-tmf 1866-espw-gb	1
1897-fmc-aroy 1884-fmc-aap	0.582484
1883-espw-btg 1911-hj-to	0.500268
1862-hbs-tpooi 1875-hbs-waon	0.664699
1883-fmc-ars 1885-fmc-tcotk	0.418053
1886-hbs-tsw 1854-hbs-smofl-v2	0.664699
1903-ew-ivatg 1878-cdw-hikab	0.637294
1858-wta-pap 1897-haj-fof	0.88394
1902-hj-twotd-v2 1888-hj-tr	0.664699
1871-hbs-ofs 1888-fmc-si	0.308433
1907-ab-rm 1901-hj-tsf	0.774319
1882-haj-ffbts 1893-haj-ianwatgfoa	0.116597
1908-rwc-tfl 1910-rwc-ap	0.88394
