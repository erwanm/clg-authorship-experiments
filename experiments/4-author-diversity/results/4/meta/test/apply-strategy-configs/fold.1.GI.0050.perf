0.300456	0.577800	0.520000
