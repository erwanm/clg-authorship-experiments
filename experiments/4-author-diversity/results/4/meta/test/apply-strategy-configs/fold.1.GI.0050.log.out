apply-config.sh: applying config '/tmp/expe4/4/process/selected-meta-configs/fold.1.0095.model/strategy-configs/fold.1.GI.0050' to cases '/home/moreaue/dump/expe4//results/4/meta/test/apply-strategy-configs/only-cases.tmp.P4VgSSjU'; results written to '/home/moreaue/dump/expe4//results/4/meta/test/apply-strategy-configs/fold.1.GI.0050'
apply-config.sh: done.
